src/hierarchies/priors/mnig_prior_model.h

```diff
@@ -10,6 +10,11 @@
 #include "hyperparams.h"
 #include "src/utils/rng.h"
 
+//! A conjugate prior model for the scalar linear regression likelihood, that
+//! is
```

Comment:
This newline is ridiculous, I'd replace `that is` with `:` and call it a day

Reply:
Damned clang format. Thanks for the suggestion